src/accessibility.md

```diff
@@ -60,31 +139,45 @@ DAC tested a sample of pages to cover the different content types in the GOV.UK
+
+DAC tested the [Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/), which is the platform that runs the GOV.UK Frontend documentation website.
 
-The [GOV.UK Frontend documentation website](http://frontend.design-system.service.gov.uk/) was last tested for accessibility issues in March 2021.
+To learn more, read the [Technical Documentation Template accessibility statement](https://tdt-documentation.london.cloudapps.digital/accessibility/).
 
-### How the GOV.UK Design System team makes this website accessible
+#### Frontend codebase
 
-The GOV.UK Design System team works hard to ensure that this Design System and Frontend, the codebase it uses, are accessible.
+The GOV.UK Frontend codebase is not a website but a [code repository](https://github.com/alphagov/govuk-frontend) and [npm package](https://www.npmjs.com/package/govuk-frontend).
+
+Read more about the codebase’s accessibility in the [GOV.UK Frontend GitHub repository readme](https://github.com/alphagov/govuk-frontend#accessibility).
+
+### How the GOV.UK Design System team makes their websites accessible
+
+The GOV.UK Design System team works to make sure the Design System and Frontend websites, and the codebase they use, are accessible.
 
 Where possible, the team aims to research components and patterns with a representative range of users, including those with disabilities.
 
-We also test components to ensure they work with a broad range of browsers, devices and assistive technologies - including screen magnifiers, screen readers and speech recognition tools.
+We also test components to make sure they work with a broad range of browsers, devices and assistive technologies - including screen magnifiers, screen readers and speech recognition tools.
 
-When we publish new content, we'll continue to make sure that it meets accessibility standards.
+When we publish new content, we’ll continue to make sure that it meets accessibility standards.
 
 ### Preparation of this accessibility statement
 
-This statement was prepared on 23 October 2019. It was last reviewed on 30 June 2021.
+This statement was prepared on 23 October 2019. It was last reviewed on 26 April 2023.
 
 ## Using the Design System and Frontend in your service
 
 While the GOV.UK Design System team takes steps to ensure the Design System is as accessible as possible by default, you still need to carry out contextual research.
 
-Using Frontend does not mean your service automatically meets level AA of WCAG 2.1. You’ll still need to make sure your service as a whole meets accessibility requirements.
+Using only the code supplied in the GOV.UK Frontend codebase is not enough to accessibly implement the GOV.UK Design System. There is important written guidance in the [GOV.UK Design System website](https://design-system.service.gov.uk/) on how to implement styles, components and patterns in an accessible way. 
```

Comment:
Might be clearer to say it this another way, so there isn't the other interpretation of "only".

```suggestion
Simply using code from the GOV.UK Frontend codebase is not enough to accessibly implement the GOV.UK Design System. There is important written guidance in the [GOV.UK Design System website](https://design-system.service.gov.uk/) on how to implement styles, components and patterns in an accessible way. 
```